lib/dynflow/web/console.rb

```diff
@@ -50,16 +50,14 @@ class Console < Sinatra::Base
       end
 
       post('/worlds/check') do
-        load_worlds
         @validation_results = world.worlds_validity_check(params[:invalidate])
-        @worlds = world.coordinator.find_worlds
+        load_worlds
```

Comment:
Was thre any reason for this change? Just trying to understand what's the reasoning here